src/explore-education-statistics-frontend/src/modules/data-catalogue/DataCataloguePage.tsx

```diff
@@ -66,64 +639,51 @@ export const getServerSideProps: GetServerSideProps<Props> = async context => {
     }),
   );
 
-  const selectedPublication = themes
-    .flatMap(option => option.topics)
-    .flatMap(option => option.publications)
-    .find(option => option.slug === publicationSlug);
-
-  let releases: ReleaseSummary[] = [];
-  let selectedRelease: ReleaseSummary | undefined;
-
-  if (selectedPublication) {
-    releases = await queryClient.fetchQuery(
-      publicationQueries.listReleases(selectedPublication.slug),
-    );
-
-    selectedRelease = releaseSlug
-      ? releases.find(rel => rel.slug === releaseSlug)
-      : undefined;
+  // Redirect old slug based links to new format.
+  if (publicationSlug) {
+    const selectedPublication = themes
+      .flatMap(option => option.topics)
+      .flatMap(option => option.publications)
+      .find(option => option.slug === publicationSlug);
 
-    // Redirect old slug based links to new format.
-    if (newDesign) {
-      const redirectUrlQuery = selectedRelease
-        ? `publicationId=${selectedPublication.id}&releaseId=${selectedRelease.id}`
+    if (selectedPublication) {
```

Comment:
I wonder if it's worth being a bit stricter about this - If you put in any old garbage in place of the slugs this will evaluate `selectedPublication` as undefined, fall through and display the page as if you'd just visited `./data-catalogue` without the slugs. This will be a pain point for SEO at the very least because we're serving duplicated content for two URLs, but also Google could end up indexing it too leading to more redirects. I'd recommend we serve a 404 in this case (or redirect to `./data-catalogue`, but then that raises the question of why we don't do that elsewhere for other routes). 

Finally, I tried this out on the live site and realised we have the same problem, so apologies for flagging something you haven't introduced but it would be nice to sort it :)

Reply:
Good point! I've set it to redirect to `/data-catalogue` instead of 404ing as it feels more user friendly, and hopefully prevent us having to set up specific redirects for urls that 404 later on.